pkg/api/unversioned/group_version.go

```diff
@@ -241,6 +253,28 @@ func (gv *GroupVersion) UnmarshalText(value []byte) error {
 	return gv.unmarshal(value)
 }
 
+// GroupVersions can be used to represent a set of desired group versions.
+type GroupVersions []GroupVersion
+
+// VersionForGroup implements the same interface as GroupVersions to allow easy
```

Comment:
huh?  I don't thinks this godoc makes sense.  This is *on* the GroupVersions type, you can't delegate your godoc to ti.